### PR TITLE
Fix part of #3602: Added label for ProfilePictureActivity

### PR DESCRIPTION
**Pull request by deepanshu731**
Fix part of #3602  Added label for ProfilePictureActivity. 
Screenshot from 2021-10-12 16-37-26

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -307,6 +307,8 @@
   <string name="profile_edit_delete_dialog_negative">Cancel</string>
   <string name="profile_edit_allow_download_heading">Allow Download Access</string>
   <string name="profile_edit_allow_download_sub">User is able to download and delete content without Administrator password</string>
+  <!-- ProfilePictureActivity -->
+  <string name="profile_picture_activity_title">profile picture activity</string>
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: Update to <string name="profile_picture_activity_title">Profile Picture</string>

**Review by rt4914 — CHANGES_REQUESTED** [review]
@deepanshu731 PTAL. Also please add screenshot displaying test case is passing on Espresso. (It is mentioned as last point in your PR description.)

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (deepanshu731) [reply app/src/main/res/values/strings.xml]: done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfilePictureActivityTest.kt [hunk 2]
```diff
@@ -92,6 +94,11 @@ class ProfilePictureActivityTest {
   @get:Rule
   val accessibilityTestRule = AccessibilityTestRule()
 
+  @get:Rule
+  val activityTestRule: ActivityTestRule<ProfilePictureActivity> = ActivityTestRule(
```
Comment (BenHenning) [inline app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfilePictureActivityTest.kt]: This rule is actually deprecated. Can we use ActivityScenario or ActivityScenarioRule, instead? There are a bunch of tests that use these two patterns for reference.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @deepanshu731. Just had one follow-up comment, PTAL.

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfilePictureActivityTest.kt (hunk 2 above)
Comment (deepanshu731) [reply app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfilePictureActivityTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfilePictureActivityTest.kt [hunk 3]
```diff
@@ -123,6 +125,20 @@ class ProfilePictureActivityTest {
     )
   }
 
+  @Test
+  fun testProfilePictureActivity_hasCorrectActivityLabel() {
+    val scenario: ActivityScenario<ProfilePictureActivity> = launch(
+      ProfilePictureActivity::class.java
+    )
```
Comment (BenHenning) [inline app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfilePictureActivityTest.kt]: We should use ``use`` here, like so:

```kotlin
launch(ProfilePictureActivity::class.java).use { scenario ->
  scenario.onActrivity { activity ->
...
```

This ensures the scenario is 'closed' (which shuts down the activity in Espresso tests).

**Review by BenHenning — COMMENTED** [review]
Thanks @deepanshu731! The new test looks good, just had one other comment. PTAL.

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfilePictureActivityTest.kt (hunk 3 above)
Comment (deepanshu731) [reply app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfilePictureActivityTest.kt]: done

**Review by BenHenning — APPROVED** [review]
Thanks @deepanshu731! LGTM.

**Comment by BenHenning** [comment]
Merging since all conversations are resolved & all approvals are present.

**Pull request merged**